eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -189,6 +194,13 @@ object Commitments {
       return Failure(HtlcValueTooSmall(commitments.channelId, minimum = htlcMinimum, actual = cmd.amount))
     }
 
+    // we allowed mismatches between our feerates and our remote's as long as commitments didn't contain any HTLC at risk
+    // we need to verify that we're not disagreeing on feerates anymore before offering new HTLCs
+    val localFeeratePerKw = feeConf.feeEstimator.getFeeratePerKw(target = feeConf.feeTargets.commitmentBlockTarget)
```

Comment:
This error should be translated to a `TemporaryChannelFailure` at relay.

Reply:
Good point, I'll write some tests and make sure this is the case.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1473/commits/e03d05793ba4f402b1f1c2ab4c61ae92408bacd5